.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we move this config inside `playground` folder and maybe remove `babel.config.js` which is something `wp-scripts build/start` will use in the same form regardless?

Reply:
There is no plan on supporting `babel.config.js` with Parcel:
https://github.com/parcel-bundler/parcel/issues/2110#issuecomment-457943670

Reply:
Moved :)